maker/src/routes.rs

```diff
@@ -300,7 +304,12 @@ pub async fn get_cfds<'r>(
     let rx_cfds = rx.cfds.clone();
     let cfds = rx_cfds.borrow().clone();
 
-    Ok(Json(cfds))
+    match cfds {
+        Some(cfds) => Ok(Json(cfds)),
+        None => Err(HttpApiProblem::new(StatusCode::INTERNAL_SERVER_ERROR)
```

Comment:
Maybe `StatusCode::SERVICE_UNAVAILABLE` fits better here? 

https://www.rfc-editor.org/rfc/rfc7231#section-6.6.4

Reply:
I like the idea of returning a code, as well as the suggestion of returning `Retry-After! :)

Reply:
If you guys don't mind, I would propose doing it in a follow-up. Things to be considered are:

- What should the value of `Retry-After` be?
- How do we see this being used in clients? Most of our clients fetch this on an interval anyway. Would we expect them to honor the value in `Retry-After` and dynamically offset the retry interval?

I like the idea in principal but I am just not sure if it is worth it right now.

Reply:
> I like the idea in principal but I am just not sure if it is worth it right now.

no need to do this now. Also, we would need to implement that logic in the client to make use if it ;)